locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -214,6 +215,20 @@ const getAttachments = async ({
     }
   }
 
+  // Add certificate when lock is certificate
+  if (isCertification && network && tokenId) {
+    const certificate = await createCertificate({
+      network,
+      lockAddress,
+      tokenId,
+    })
+    const svg = new resvg.Resvg(certificate)
+    const pngData = svg.render()
+    const pngBuffer = pngData.asPng()
+    const dataURI = `data:image/png;base64,${pngBuffer.toString('base64')}`
```

Comment:
Why do you need to render as png and not use svg since you are doing a data uri anyway?

Reply:
refactored, but still i think we need to keep it as is, the SVG need to be decoded otherwise

i think @searchableguy  can give more details about the implementation here

Reply:
I'm confused. If you are talking about the reason why I convert svg to png for tickets, that's for displaying the ticket in emails and make it easy to print.